### PR TITLE
Task 6

### PR DESCRIPTION
**Pull request by ZiaurRehmanmalhi**

### CODE REVIEW

File: task/task-1.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 from utils.reader import read_file
-
-list_data = read_file()
-for data in list_data:
+file_read_csv = read_file("/home/zia/ISDP/Hammad sab/Python/wether_Mar/weather task 2 in csv file/files/f1.csv")
```
Comment (Hamma111) [inline task/task-1.py:4]: always 2 new lines space after imports. this mistake should not happen again.

File: task/task-2.py [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
+file_read_csv = read_file("/home/zia/ISDP/Hammad sab/Python/wether_Mar/weather task 2 in csv file/files/f1.csv")
 
-list_data = read_file()
-for data in list_data:
+for data in file_read_csv:
     date = data.split(",")[0]
     max_temp = data.split(",")[1]
     min_temp = data.split(",")[3]
-    avrage = (int(max_temp) - int(min_temp))/2
+    avrege = (int(max_temp) - int(min_temp))/2
```
Comment (Hamma111) [inline task/task-2.py]: average*

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: task/task-1.py (hunk 1 above)
Comment (ZiaurRehmanmalhi) [reply task/task-1.py:4]: ok Inshallah

File: task/task-2.py (hunk 2 above)
Comment (ZiaurRehmanmalhi) [reply task/task-2.py]: ok i will change

**Pull request merged**